test/integration/smart_answer_flows/energy_grants_calculator_test.rb

```diff
@@ -629,9 +644,10 @@ class EnergyGrantsCalculatorTest < ActiveSupport::TestCase
     # test for incomesupp_jobseekers_2?
     context "test that incomesupp_jobseekers_2? is being calculated correctly at Q4" do
       setup do
+        Timecop.travel("2016-07-01")
         add_response 'all_help'
         add_response 'benefits'
-        add_response '08/06/1952'
+        add_response '08/06/1953'
```

Comment:
Can you update the commit note to explain why you've had to use Timecop and change this date, please? Presumably it's something to do with `08/06/1952` being earlier than `winter_fuel_payment_threshold` which then changes the benefits someone is eligible for?